cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -46,7 +46,8 @@ type MasterConfiguration struct {
 	// SelfHosted enables an alpha deployment type where the apiserver, scheduler, and
 	// controller manager are managed by Kubernetes itself. This option is likely to
 	// become the default in the future.
-	SelfHosted bool
+	SelfHosted     bool
+	SkipTokenPrint bool
```

Comment:
That would make the func signatures have more args to pass in. Just like when we moved `selfhosted` to the api instead of an arg, I argue this is a similar case.